mozsvc/tests/test_user.py

```diff
@@ -30,9 +30,12 @@
     from mozsvc.user.noncecache import MemcachedNonceCache
     # We'll test for a live memcached server when we actually run the tests.
     MEMCACHED = None
-except (ImportError, BackendError):
+except (ImportError, BackendError) as err:
+    print("Import of memcached classes failed: {}".format(err))
```

Comment:
Is this `print` just for debugging or should we keep it in the final merge?

Reply:
This was more used for debugging.
I changed the message that will be hit when the import failed (line 329) to make understanding the problem easier.